doc/examples/jest+react/package.json

```diff
@@ -8,21 +8,21 @@
     "organization": "Deque Systems, Inc.",
     "url": "http://deque.com/"
   },
-  "dependencies": {
-  },
+  "dependencies": {},
   "scripts": {
     "test": "jest"
   },
   "devDependencies": {
-    "axe-core": "^2.2.3",
-    "babel-jest": "^20.0.3",
+    "axe-core": "^2.6.1",
```

Comment:
Should we update these to axe 3x since they're going onto develop?

Reply:
I didn't want to point to a beta or unreleased version unless I had to. The example doesn't have any shadow DOM. We can update this when we release 3.0.